lib/presentation/pages/intro_page.dart

```diff
@@ -68,7 +81,7 @@ class _IntroPageState extends State<IntroPage> {
           PageViewModel(
             titleWidget: const SizedBox(),
             decoration: emptyPageDecoration(),
-            bodyWidget: TipsOrganism(studyType),
+            bodyWidget: TipsOrganism(studyType, dndEnabled ?? false),
```

Comment:
I understand where this change coming from but it is better to keep the code responsible for the page and not passing it on (maybe if it is long operations which it is not).

We might also change the check dnd to a listener in the future to change the state live while the user toggle it from the settings